privaterelay/templates/profile.html

```diff
@@ -53,18 +54,6 @@
   </div>
   {% endif %}
   <div class="main-list flx flx-col jst-cntr">
-  {% if messages %}
-    <div class="messages hidden">
-        {% for message in messages %}
-        <div{% if message.tags %} class="{{ message.tags }} js-notification" {% else %} class="" {% endif %}>
-          <span>{{ message }}</span>
-          <button class="dismiss">
-            <svg class="x-close-icon fill-current text-gray hover:text-purple-800" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z"/></svg>
-          </button>
-        </div>
-      {% endfor %}
-    </div>
-  {% endif %}
```

Comment:
I realized this `div` should not have been here, this entire snippet belongs to the `messages` partial that's already present at line 20. The `hidden` class was what was keeping it invisible.